bndtools.builder/src/org/bndtools/builder/BndtoolsBuilder.java

```diff
@@ -117,6 +119,7 @@
             if (model == null) {
                 try {
                     model = Central.getProject(myProject.getLocation().toFile());
+                    model.addBasicPlugin(new JobProgress());
```

Comment:
The plugins are a set and you keep adding a new one every time a project is processed by the builder. This seems like a memory leak.